dir.props

```diff
@@ -69,14 +69,16 @@
   <!-- Provides package dependency version properties and verification/auto-upgrade configuration -->
   <Import Project="$(MSBuildThisFileDirectory)dependencies.props" />
 
-  <!-- Explicitly setting
-       - SupportsUWP to true when TargetGroup is uap101*. This will enable the correct pinvoke checker to be picked
-       - BlockReflectionAttribute to true. This will enable injecting BlockReflectionAttribute.cs into the compile target.
-       TODO: Both of these properties are being set in buildtools targets, however they need to be updated to reflect uap101 changes.
-  -->
-  <PropertyGroup Condition="'$(TargetGroup)' == 'uap101' or '$(TargetGroup)' == 'uap101aot'">
+  <!-- Only if we're building a binary flavor that's only ever going to run on uapaot is OK for it to have #if NETNATIVE blocks -->
+  <PropertyGroup Condition="'$(TargetGroup)' == 'uapaot'">
+    <DefineConstants>$(DefineConstants),NETNATIVE</DefineConstants>
```

Comment:
It would think it should be just `#if uap` in most places. `#if uap || uapaot` should be rare. It depend on the build configurations.

uapaot build configuration is specialization of uap build configuration. The uap build configuration is used to satisfy uapaot vertical in many places. For example, 
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/Configurations.props has just uap build configuration that will be used for both uap and uapaot vertical. (@weshaggard please correct me if I am wrong.)